internal/blockchain/ethereum/quorum/geth_provider.go

```diff
@@ -0,0 +1,409 @@
+		}
+	}
+	serviceDefinitions = append(serviceDefinitions, p.connector.GetServiceDefinitions(p.stack, connectorDependents)...)
+	return serviceDefinitions
+}
+
+func (p *GethProvider) GetBlockchainPluginConfig(stack *types.Stack, m *types.Organization) (blockchainConfig *types.BlockchainConfig) {
+	var connectorURL string
+	if m.External {
+		connectorURL = p.GetConnectorExternalURL(m)
+	} else {
+		connectorURL = p.GetConnectorURL(m)
+	}
+
+	blockchainConfig = &types.BlockchainConfig{
+		Type: "ethereum",
+		Ethereum: &types.EthereumConfig{
+			Ethconnect: &types.EthconnectConfig{
+				URL:   connectorURL,
+				Topic: m.ID,
+			},
+		},
+	}
+	return
+}
+
+func (p *GethProvider) GetOrgConfig(stack *types.Stack, m *types.Organization) (orgConfig *types.OrgConfig) {
+	account := m.Account.(*ethereum.Account)
+	orgConfig = &types.OrgConfig{
+		Name: m.OrgName,
+		Key:  account.Address,
+	}
+	return
+}
+
+func (p *GethProvider) Reset() error {
+	return nil
+}
+
+func (p *GethProvider) GetContracts(filename string, extraArgs []string) ([]string, error) {
+	contracts, err := ethereum.ReadContractJSON(filename)
+	if err != nil {
+		return []string{}, err
+	}
+	contractNames := make([]string, len(contracts.Contracts))
+	i := 0
+	for contractName := range contracts.Contracts {
+		contractNames[i] = contractName
+		i++
+	}
+	return contractNames, err
+}
+
+func (p *GethProvider) DeployContract(filename, contractName, instanceName string, member *types.Organization, extraArgs []string) (*types.ContractDeploymentResult, error) {
+	contracts, err := ethereum.ReadContractJSON(filename)
+	if err != nil {
+		return nil, err
+	}
+	return p.connector.DeployContract(contracts.Contracts[contractName], instanceName, member, extraArgs)
+}
+
+func (p *GethProvider) CreateAccount(args []string) (interface{}, error) {
+	l := log.LoggerFromContext(p.ctx)
+	memberIndex := args[2]
+	memberCount := args[3]
+	gethVolumeName := fmt.Sprintf("%s_geth_%s", p.stack.Name, memberIndex)
+	var directory string
+	stackHasRunBefore, err := p.stack.HasRunBefore()
+	if err != nil {
+		return nil, err
+	}
+	if stackHasRunBefore {
+		directory = p.stack.RuntimeDir
+	} else {
+		directory = p.stack.InitDir
+	}
+
+	prefix := strconv.FormatInt(time.Now().UnixNano(), 10)
+	outputDirectory := filepath.Join(directory, "blockchain", fmt.Sprintf("geth_%s", memberIndex), "keystore")
+	keyPair, walletFilePath, err := ethereum.CreateWalletFile(outputDirectory, prefix, keyPassword)
+	if err != nil {
+		return nil, err
+	}
+
+	// Tessera is an optional add-on to the quorum blockchain node provider
+	var tesseraPubKey, tesseraKeysPath string
+	if p.stack.TesseraEnabled {
+		tesseraVolumeName := fmt.Sprintf("%s_tessera_%s", p.stack.Name, memberIndex)
+		tesseraKeysOutputDirectory := filepath.Join(directory, "tessera", fmt.Sprintf("tessera_%s", memberIndex), "keystore")
+		_, tesseraPubKey, tesseraKeysPath, err = CreateTesseraKeys(p.ctx, tesseraImage, tesseraKeysOutputDirectory, "", "tm", keyPassword)
```

Comment:
what does the name `tm` signify to a user for the new keys?

Reply:
tm signifies transaction manager. It's the default used in most of consensys's examples, ref: https://github.com/Consensys/quorum-dev-quickstart/blob/master/files/common/config/tessera/data/tessera-config-template.json